internal/events/eventstream.go

```diff
@@ -132,9 +132,7 @@ func NewEventStream(
 		wsChannels:         wsChannels,
 		retry:              esDefaults.retry,
 		checkpointInterval: config.GetDuration(tmconfig.EventStreamsCheckpointInterval),
-	}
-	if config.GetInt(tmconfig.ConfirmationsRequired) > 0 {
-		es.confirmations = confirmations.NewBlockConfirmationManager(esCtx, connector)
+		confirmations:      confirmations.NewBlockConfirmationManager(esCtx, connector),
```

Comment:
I don't remember us looking to remove this here. I know we discussed doing that the `Manager` would use confirmation manager even if `ConfirmationsRequired`, because the receipt checking is in the confirmations manager for that path. But here I thought we were going to leave the bypass in place.

Reply:
With the check here, if confirmations was set to zero, it would never create the confirmation manager at all. This change makes it always create a confirmation manager, regardless of what confirmations is set to, which is what I thought we discussed.